bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss

```diff
@@ -0,0 +1,42 @@
+:root {
+  --color-transparent: #ff000000;
```

Comment:
Could probably move this to `/imports/ui/stylesheets/variables/palette.scss`

Reply:
moved